src/main/java/TaskList.java

```diff
@@ -0,0 +1,124 @@
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+public class TaskList {
+
+    private ArrayList<Task> tasks = new ArrayList<>();
+
+    public enum TaskType {
+        DEADLINE,
+        EVENT,
+        TODO
+    }
+
+    public void addToList(TaskType taskType, String description, String deadline)
```

Comment:
Maybe make this method take in a `Task` and construct the `Task` at the call site. This reduces duplicate code.